modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * @prettier
  */
-import { Codes, VirtualSizes } from '@bitgo/unspents';
+import { Codes } from '@bitgo/unspents';
 import { UnspentType } from '@bitgo/unspents/dist/codes';
```

Comment:
Hm, do we need to pull this out of `dist/codes`? Is it not exported? (Optional nit since this code is unchanged)